.github/workflows/benchmark.yml

```diff
@@ -51,28 +53,28 @@ jobs:
              pip3 install --upgrade pip PyYAML setuptools redisbench-admin
              pip3 install -r requirements.txt
 
-      - name: stress test
-        run: |
-          sudo apt install stress-ng -qqy
-          uptime
-          stress-ng -c 1 --timeout 60s --metrics-brief
-          uptime
-          stress-ng --stream 1 -t 60 --metrics-brief
-          uptime
-          stress-ng --ipsec-mb=1 -t 60 --metrics-brief
-          uptime
+      # - name: stress test
+      #   run: |
+      #     sudo apt install stress-ng -qqy
+      #     uptime
+      #     stress-ng -c 1 --timeout 60s --metrics-brief
+      #     uptime
+      #     stress-ng --stream 1 -t 60 --metrics-brief
+      #     uptime
+      #     stress-ng --ipsec-mb=1 -t 60 --metrics-brief
+      #     uptime
```

Comment:
What is that and why comment out?

Reply:
a step that Filipe left here, he wanted to add it to the graphs/normalize them according to these results. Until then, let's comment them out.

Reply:
OK, @filipecosta90 any updates regarding this?